src/modules.test.ts

```diff
@@ -0,0 +1,106 @@
+/* tslint:disable only-arrow-functions */
+import { jaspi3xUrl, removeRequire, stubRequire } from '../test/helpers';
```

Comment:
jaspi3xUrl is misspelled.

(also, a totally uppercase `JSAPI3XURL` constant would look a little nicer).